src/app/header/header.component.scss

```diff
@@ -8,14 +27,14 @@ a {
   color: white;
 }
 
-.mat-chip-list a {
-  color: black;
+.mat-form-field {
+  width: 100%;
 }
 
-.mat-form-field {
-  width: 400px;
+::ng-deep .mat-form-field-appearance-fill .mat-form-field-flex {
+  background-color: rgba(0, 0, 0, 0) !important;
```

Comment:
Is the !important really needed?
In most cases its enough to add a css class to the path.
See for more info:

https://www.w3docs.com/snippets/css/how-to-override-css-styles.html

Reply:
removed the !important